lib/urlwatch/jobs.py

```diff
@@ -196,7 +197,10 @@ def ignore_error(self, exception):
 
 class Job(JobBase):
     __required__ = ()
-    __optional__ = ('name', 'filter', 'max_tries', 'diff_tool', 'compared_versions', 'diff_filter', 'enabled', 'treat_new_as_changed', 'user_visible_url')
+    __optional__ = ('name', 'tags', 'filter', 'max_tries', 'diff_tool', 'compared_versions', 'diff_filter', 'enabled', 'treat_new_as_changed', 'user_visible_url')
+
+    def matches_tags(self, tags: Set[str]):
+        return self.tags & tags
```

Comment:
Is it `matches_tags` or `matching_tags`? Could have a type annotation for the return type?